module/setup/deprecations.js

```diff
@@ -27,7 +27,7 @@ async function globalModifiers() {
 async function specialModifiers() {
     let affectedActors = game.actors
         .filter(actor => actor.isOwner)
-        .filter(actor => actor.system.specialSkillModifiers.length > 0);
+        .filter(actor => actor.system.specialSkillModifiers && actor.system.specialSkillModifiers.length > 0);
```

Comment:
An empty array is a falsy value. So this should not be necessary

Reply:
> An empty array is a falsy value. So this should not be necessary

This error appears. Is it possible to remove this error log in another way?
![image](https://github.com/user-attachments/assets/484f793a-fd94-45e1-9b29-26b58f7bcd6d)